src/main/java/org/opensearch/flowframework/common/CommonValue.java

```diff
@@ -53,6 +53,8 @@ private CommonValue() {}
     public static final String WORKFLOW_URI = FLOW_FRAMEWORK_BASE_URI + "/workflow";
     /** Field name for workflow Id, the document Id of the indexed use case template */
     public static final String WORKFLOW_ID = "workflow_id";
+    /** Field name for dry run, the flag to indicate if validation is necessary */
+    public static final String DRY_RUN = "dryrun";
```

Comment:
Nit: Should we call it validate? No Strong opinion here.

Reply:
Alerting plugin uses `dryrun=true` in the URI: https://opensearch.org/docs/latest/observing-your-data/alerting/api/ and this is the closest example to our usage.

Ingesting a data stream uses a `dry_run` field inside the body: https://opensearch.org/docs/latest/im-plugin/data-streams/

Several REST paths include `validate` or `_validate` as part of the path but not as a param.